packages/datadog-plugin-nats/src/consumer.js

```diff
@@ -0,0 +1,100 @@
+'use strict'
+
+const ConsumerPlugin = require('../../dd-trace/src/plugins/consumer')
+const Extractors = require('./extractors')
```

Comment:
Things that aren't classes shouldn't have a leading capital letter. This is not the only place in this PR where this happens.